sphinx_rtd_theme/versions.html

```diff
@@ -7,6 +7,7 @@
       <span class="fa fa-caret-down"></span>
     </span>
     <div class="rst-other-versions">
+      {% if theme_versions_menu %}
```

Comment:
Clean up the whitespace with `{%- if theme_versions_menu %}` else we get 20 blank lines if this is false

http://jinja.pocoo.org/docs/2.10/templates/#whitespace-control